app/client/src/common/components/Page404.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @flow
+ */
+
+import React from 'react'
+
+function Page404() {
+    return (
+        <h1>
+            Error 404
+        <h1 />
+        <h2>
+            Oops, we are sorry but the page you are looking for was not found!
+            <br>
+            Maybe you are looking for something that doesn't exist
+        <h2 />
+    )
+}
```

Comment:
A React component can only render one top level element. In this case you are trying to render 2 siblings together as top level elements. The old way to handle this situation was to wrap your elements in one top-level parent element:

```jsx
import React from 'react'

function Page404() {
  return (
    <div>
      <h1>Error 404</h1>
      <h2>
        Oops, we are sorry but the page you are looking for was not found!
        <br />
        Maybe you are looking for something that doesn't exist
      </h2>
    </div>
  )
}

export default Page404
```

But this creates an unnecessary HTML `div` element that we don't really need. Ever since React 16, we can use `Fragment` instead to let a component render multiple elements together without introducing an extra HTML element:

```jsx
import React, { Fragment } from 'react'

function Page404() {
  return (
    <Fragment>
      <h1>Error 404</h1>
      <h2>
        Oops, we are sorry but the page you are looking for was not found!
        <br />
        Maybe you are looking for something that doesn't exist
      </h2>
    </Fragment>
  )
}

export default Page404
```